### ISSUE TITLE
Bugged cyborgs docking stations

### ISSUE THREAD

**Issue by goonstation-issuebot** [goonstation/goonstation#2249]
Submitter: eternalmuerto
**Describe the bug**
AI died, then been rebuilded, but now cyborg docking stations can be use only if no one inside

**To Reproduce**
1. AI died
2. AI was rebuild
3. Cyborgs docking stations shows it's interface only if no borgs inside

**Expected behavior**
Standart interface of docking station (to repair myself after explosion)

**Additional context**
Humans can't interract with stations too when someone inside, but all works if empty

Reported on: Goonstation Roleplay 2020-09-23 06:52:25

**Comment by zjdtmkhzt** [comment]
Yup, I also checked in in that round as a mentor mouse,
The borg in question reported they were unable to use their left two slots or select them (they were empty).
Their rightmost slot had an automender in it, which they said was functional.

Cyborg Docking Stations that the borg was in could not have their UI opened.
After the borg left them, they seemed normal again.
I opened the UI of a docking station and then put the borg in and tried to press the refresh button in the UI, but that also did nothing.

Their battery was full and their two arms seemed to be damaged, but still had health left. (5/25)
